libs/ui-watt/src/lib/components/table/watt-table.component.scss

```diff
@@ -21,6 +21,7 @@ watt-table .mat-table {
   display: grid;
   position: relative;
   overflow: auto;
+  min-height: 45px; /* Magic number to prevent table shrinking below header height */
```

Comment:
Suggestion: Whould it be possible to have a css variable, that defines the height of the header, and is also used here?

Reply:
It is a little weird since it is basically 44px + 1px border on the th elements. I found a different solution using box-shadow, meaning I can use the same variable of 44px to set the min-height.